Products/CMFPlone/controlpanel/browser/redirects.py

```diff
@@ -190,20 +190,34 @@ def __init__(self, query="", created="", manual=""):
             else:
                 manual = ""
         if created:
+            end = created
+            logger.warning(
+                "The 'datetime' parameter is deprecated. Use 'end' parameter instead."
+            )
```

Comment:
Instead of this, do:
```suggestion
            warnings.warn(
                "The 'created' parameter is deprecated. Use 'start' parameter instead.",
                DeprecationWarning
            )
```

This makes it easier to filter out the warning if someone wants to. You also have to import warnings at the top of the file.